lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -79,8 +79,8 @@ CHECK_EXISTING_IDENTITY:
     reuse:
       targetState: IPV_IDENTITY_REUSE_PAGE
       checkFeatureFlag:
-        deleteDetailsEnabled:
-          targetState: IPV_IDENTITY_REUSE_PAGE_TEST
+        ticfCriBeta:
```

Comment:
I'm unsure if it's the convention, but I think a more informative flag names could explicitly specify that it enables the ticf cri to be called, which might be a bit obscured behind the term "beta" otherwise